notebooks/pytorch_sparse_quantized_transfer_learning.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<ul><li>Remove the mention of recalibrated, sub in something along the lines of transfer learning.</li><li>Let's rephrase to not make the ONNX export a requirement. Should put something along the lines of exporting to ONNX is a common next step for deploying the model and is the format required by the DeepSparse Engine.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>yeah, a quick mention of DeepSparse Engine being the best way to recognize the speedup from sparse quantization</p>